document/src/main/kotlin/com/ritense/document/service/CaseTagService.kt

```diff
@@ -0,0 +1,150 @@
+package com.ritense.document.service
+
+import com.ritense.authorization.Action
+import com.ritense.authorization.AuthorizationService
+import com.ritense.authorization.request.EntityAuthorizationRequest
+import com.ritense.document.domain.CaseTag
+import com.ritense.document.domain.CaseTagId
+import com.ritense.document.exception.CaseTagAlreadyExistsException
+import com.ritense.document.exception.CaseTagNotFoundException
+import com.ritense.document.repository.CaseTagRepository
+import com.ritense.document.web.rest.dto.CaseTagCreateRequestDto
+import com.ritense.document.web.rest.dto.CaseTagUpdateRequestDto
+import com.ritense.valtimo.contract.annotation.SkipComponentScan
+import jakarta.validation.Valid
+import org.springframework.stereotype.Service
+import org.springframework.transaction.annotation.Transactional
+import org.springframework.validation.annotation.Validated
+import kotlin.jvm.optionals.getOrNull
+
+@Validated
+@Transactional
+@Service
+@SkipComponentScan
+class CaseTagService(
+    private val caseTagRepository: CaseTagRepository,
+    private val documentDefinitionService: DocumentDefinitionService,
+    private val authorizationService: AuthorizationService
+) {
+
+    fun getCaseTags(documentDefinitionName: String): List<CaseTag> {
+        return caseTagRepository.findByIdCaseDefinitionNameOrderByOrder(documentDefinitionName)
+    }
+
+    fun get(caseDefinitionName: String, caseTagKey: String): CaseTag {
+        return caseTagRepository.getReferenceById(CaseTagId(caseDefinitionName, caseTagKey))
+    }
+
+    fun exists(caseDefinitionName: String, caseTagKey: String): Boolean {
+        return caseTagRepository.existsByIdCaseDefinitionNameAndIdKey(caseDefinitionName, caseTagKey)
+    }
+
+    fun create(
+        caseDefinitionName: String,
+        @Valid request: CaseTagCreateRequestDto) : CaseTag{
```

Comment:
When the arguments span multiple lines, we usually put the `) : CaseTag {` on a new line. Also, please use a space between the return type and brace to open the method.